src/Command/InstallProjectCommand.php

```diff
@@ -0,0 +1,213 @@
+    protected function runDropDatabase(): void
+    {
+        $process = new Process(['bin/console', 'doctrine:database:drop', '--if-exists', '--force'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to drop database ...');
+    }
+
+    /**
+     * @return void
+     */
+    protected function runCreateDatabase(): void
+    {
+        $process = new Process(['bin/console', 'doctrine:database:create'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to create database ...');
+    }
+
+    /**
+     * @return void
+     */
+    protected function runDoctrineSchemaUpdate(): void
+    {
+        $process = new Process(
+            [
+                'bin/console',
+                'doctrine:schema:update',
+                '--no-interaction',
+                '--force',
+                '-vvv',
+            ], $this->projectDir
+        );
+
+        $this->runProcessBlockCommands(
+            $process,
+            'Trying to update database schema ...',
+            'Successfully database schema created'
+        );
+    }
+
+    /**
+     * @return void
+     */
+    protected function runDoctrineFixturesLoad(): void
+    {
+        $process = new Process(
+            [
+                'bin/console',
+                'doctrine:fixtures:load',
+                '--no-interaction',
+                '-vvv',
+            ], $this->projectDir
+        );
+
+        $this->runProcessBlockCommands($process, 'Trying to load Doctrine fixtures ...');
+    }
+
+    protected function runYarnInstall(): void
+    {
+        $process = new Process(['yarn', 'install', '--check-files'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to execute yarn install ...');
+    }
+
+    protected function runYarnBuild(): void
+    {
+        $process = new Process(['yarn', 'run', 'build'], $this->projectDir);
+
+        $this->runProcessBlockCommands($process, 'Trying to execute yarn run build ...');
+    }
+
+    /**
+     * @param Process $process
+     * @param string $message
+     * @param string $successOutput
+     * @return void
+     */
+    protected function runProcessBlockCommands(Process $process, string $message, $successOutput = ''): void
+    {
+        $this->io->newLine(2);
+        $this->displayNote($message);
+
+        $process->setTimeout($this->timeout);
+
+        $this->runProcess($process);
+
+        if (empty($successOutput)) {
+            $this->displayProcessSuccessOutput($process);
+        } else {
+            $this->displaySimpleSuccessOutput($successOutput);
+        }
+
+        $this->io->newLine(2);
+    }
+
+    /**
+     * @param $message
+     */
+    protected function displayNote($message): void
+    {
+        $this->io->text('<fg=yellow;bg=black>' . $message . '</>');
+    }
+
+    /**
+     * @param Process $process
+     */
+    protected function runProcess(Process $process): void
+    {
+        $process->mustRun();
```

Comment:
not sure what's the point of this wrapping method. Unless you had some idea and no time to implement it.